com.obsproject.Studio.Plugin.NDI.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id": "com.obsproject.Studio.Plugin.NDI",
+    "runtime": "com.obsproject.Studio//stable",
+    "sdk": "org.kde.Sdk//6.5",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/app/plugins/NDI"
+    },
+    "modules": [
+        "avahi.json",
+        {
+            "name": "obs-ndi",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DENABLE_FRONTEND_API=ON",
+                "-DENABLE_QT=ON"
+            ],
+            "post-install": [
+                "install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ../${FLATPAK_ID}.metainfo.xml",
+                "appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/obs-ndi/obs-ndi.git",
+                    "tag": "4.13.0",
+                    "commit": "bdd6e2610b965f8a743063f768556e318835607a",
+                    "x-checker-data": {
+                        "type": "git",
+                        "is-main-source": true,
+                        "tag-pattern": "^([\\d.]+)$"
+                    }
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "sed -i 's|/usr|/app/plugins/NDI/extra|g' src/plugin-main.cpp"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "com.obsproject.Studio.Plugin.NDI.metainfo.xml"
+                }
+            ]
+        },
+        {
+            "name": "libndi",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D apply_extra $FLATPAK_DEST/bin/apply_extra"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "echo 'Extracting NDI library'",
```

Comment:
These lines here will mess up the progress bar in flatpak cli, I think. Do you need these?

Reply:
I can remove them.